### PR TITLE
Adding Microsoft OpenJDK 16.0.1+9

### PR DESCRIPTION
**Pull request by d3vel0per**

### CODE REVIEW

**Comment by d3vel0per** [comment]
Hi @se35710 
Please review a few PRs I have for onboarding Microsoft OpenJDK GA releases
Thanks!

File: bucket/microsoft16.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "description": "The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.",
+    "homepage": "https://www.microsoft.com/openjdk",
+    "version": "16.0.1-9",
+    "license": {
+        "identifier": "GPL2 with Classpath Exception",
```
Comment (rasa) [inline bucket/microsoft16.json:6]: Perhaps this should be `GPL-2.0-with-classpath-exception` per 
https://github.com/lukesampson/scoop/wiki/App-Manifests#optional-properties and 
https://spdx.org/licenses/ ?

Comment (rasa) [inline bucket/microsoft16.json:6]:
```suggestion
        "identifier": "GPL-2.0-only WITH Classpath-exception-2.0",
```
Per https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html

Comment (d3vel0per) [reply bucket/microsoft16.json]: Updated as per comments

**Comment by d3vel0per** [comment]
Closing PR as it has been merged into #288

**Pull request closed without merging**